packages/flutter/test/material/bottom_sheet_test.dart

```diff
@@ -45,7 +45,7 @@ void main() {
     expect(showBottomSheetThenCalled, isFalse);
   });
 
-  testWidgets('Tapping outside a modal BottomSheet should dismiss it', (WidgetTester tester) async {
+  testWidgets('Tapping outside a modal BottomSheet should dismiss it by default', (WidgetTester tester) async {
```

Comment:
We should also add a test for actively setting isDimissible=true, since this only tests the default isDismissible value is set to true.

Reply:
Done.